src/DepartmentService.Data/Interfaces/IDepartmentUserRepository.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using LT.DigitalOffice.DepartmentService.Models.Db;
+using LT.DigitalOffice.Kernel.Attributes;
+using LT.DigitalOffice.Models.Broker.Requests.Company;
+
+namespace LT.DigitalOffice.DepartmentService.Data.Interfaces
+{
+  [AutoInject]
+  public interface IDepartmentUserRepository
+  {
+    Task<bool> AddAsync(DbDepartmentUser departmentUser);
+
+    Task<bool> AddAsync(List<DbDepartmentUser> departmentUsers);
+
+    Task<DbDepartmentUser> GetAsync(Guid userId, bool includeDepartment);
+
+    List<Guid> Get(IGetDepartmentUsersRequest request, out int totalCount);
+
+    Task<List<DbDepartmentUser>> GetAsync(List<Guid> userIds);
+
+    void RemoveAsync(Guid userId, Guid removedBy);
```

Comment:
user Task

Reply:
why?

Reply:
we can use void with async methods, but it works differently in one moment. 
https://docs.microsoft.com/ru-ru/dotnet/csharp/programming-guide/concepts/async/async-return-types

Reply:
thx, done